Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -317,4 +317,37 @@ void ProcessThrottler::TimedActivity::updateTimer()
         m_timer.startOneShot(m_timeout);
 }
 
+template <typename T>
+static Vector<String> activityNames(const T& activities)
+{
+    Vector<String> names;
```

Comment:
Seems like we could reserve capacity here:
`names.reserveInitialCapacity(activities.size());`